release-notes/versioned/pulsar-2.11.0.md

```diff
@@ -7,6 +7,9 @@ sidebar_label: Apache Pulsar 2.11.0
 #### 2023-01-11
 
 ### Important notice
+
+Starting from 2.11.0, the minimal version of JDK on the server side (broker, functions, connectors) is required to be 17. See [PIP-156](https://github.com/apache/pulsar/issues/15207) for more details.
```

Comment:
```suggestion
Starting from 2.11.0, the minimal version of JDK on the server side (broker, functions, connectors) is required to be 17. See [Pulsar Runtime Java Version Recommendation](https://github.com/apache/pulsar#pulsar-runtime-java-version-recommendation) and [PIP-156](https://github.com/apache/pulsar/issues/15207) for more details.
```

Here we can give readers "how-to" info first and then show "what/why (details)" info to avoid users losing patience.

Reply:
OK. Committed.